packages/contracts-bedrock/deploy-config/devnetL1.ts

```diff
@@ -6,8 +6,31 @@ const config = {
   genesisOutput: ethers.constants.HashZero,
   historicalBlocks: 0,
   startingBlockTimestamp:
-    parseInt(process.env.L2OO_STARTING_BLOCK_TIMESTAMP, 10) || Date.now(),
+    parseInt(process.env.L2OO_STARTING_BLOCK_TIMESTAMP, 16) || Date.now(),
```

Comment:
Isn't it easier to specify as a decimal rather than in hex?

Reply:
The existing devnet specifies it as hex, happy to make a larger change if requested:
- https://github.com/ethereum-optimism/optimism/blob/9aa8049c199758b1a3656fe285f7cfc889db0be7/ops-bedrock/devnet-up.sh#L63
- https://github.com/ethereum-optimism/optimism/blob/9aa8049c199758b1a3656fe285f7cfc889db0be7/ops-bedrock/devnet-up.sh#L82

Reply:
Perhaps a fix to this would be to have two tasks:
- `genesis-l1`
- `genesis-l2`

Both of them are ran as part of `devnet-up.sh`

Reply:
Ah got it, then let's just leave as-is. Thanks for clarifying.